pkg/client/informers/informers_generated/internalversion/internalinterfaces/factory_interfaces.go

```diff
@@ -35,4 +36,5 @@ type SharedInformerFactory interface {
 	InformerFor(obj runtime.Object, newFunc NewInformerFunc) cache.SharedIndexInformer
 }
 
+// TweakListOptionsFunc is a function taking v1.ListOptions.
```

Comment:
This is pretty sterile. How about "TweakListOptions is a function that transforma a v1.ListOptions"?

Reply:
I like it. Should I squash some of the previous commits or add a new commit with this change?

Reply:
Please squash.

Reply:
Changed and squashed.